recipe-server/client/actions/opt-out-study/package.json

```diff
@@ -30,6 +30,11 @@
                     "type": "string",
                     "format": "uri",
                     "minLength": 1
+                },
+                "isEnrollmentPaused": {
+                    "description": "Whether the study is currently enrolling new users",
```

Comment:
I think this has the opposite meaning as the name and the code. The description alone makes me think that a value of `true` would enroll new users and `false` would not.